src/test/cypress/integration/exercises/quiz-exercise/QuizExerciseManagement.spec.ts

```diff
@@ -10,68 +10,46 @@ const admin = artemis.users.getAdmin();
 const courseManagementRequest = artemis.requests.courseManagement;
 
 // PageObjects
-const navigationBar = artemis.pageobjects.navigationBar;
 const courseManagement = artemis.pageobjects.courseManagement;
 const quizCreation = artemis.pageobjects.quizExercise.creation;
 
 // Common primitives
-let uid: string;
-let quizExerciseName: string;
 let course: any;
+const quizQuestionTitle = 'Cypress Quiz Exercise';
 
 describe('Quiz Exercise Management', () => {
     before('Set up course', () => {
-        uid = generateUUID();
         cy.login(admin);
         courseManagementRequest.createCourse().then((response) => {
             course = response.body;
         });
     });
 
-    beforeEach('New UID', () => {
-        quizExerciseName = 'Cypress Quiz ' + generateUUID();
-    });
-
     after('Delete Course', () => {
         courseManagementRequest.deleteCourse(course.id);
     });
 
     describe('Quiz Exercise Creation', () => {
-        let quizExercise: any;
-
-        afterEach('Delete Quiz', () => {
-            courseManagementRequest.deleteQuizExercise(quizExercise.id);
-        });
-
         beforeEach(() => {
-            beginQuizCreation();
+            cy.login(admin, '/course-management/');
+            courseManagement.openExercisesOfCourse(course.title, course.shortName);
+            cy.get('#create-quiz-button').click();
+            quizCreation.setTitle('Cypress Quiz Exercise ' + generateUUID());
         });
 
         it('Creates a Quiz with Multiple Choice', () => {
-            quizCreation.addMultipleChoiceQuestion('MC Quiz' + uid);
-            quizCreation.saveQuiz().then((quizResponse) => {
-                quizExercise = quizResponse.response?.body;
-                cy.visit('/course-management/' + course.id + '/quiz-exercises/' + quizExercise.id + '/preview');
-                cy.contains('MC Quiz' + uid).should('be.visible');
-            });
+            quizCreation.addMultipleChoiceQuestion(quizQuestionTitle);
+            saveAndVerifyQuizCreation(quizQuestionTitle);
```

Comment:
Seems like you're always using ``quizQuestionTitle`` as parameter for all calls of ``saveAndVerifyQuiyCreation``. I'd sugget removing the parameter of the method, and instead reference the ``quizQuestionTitle`` directly in ``saveAndVerifyQuiyCreation``.

Reply:
You are right. When I originally refactored it I first thought that each quiz type has to have its own specific title, but as it turned out that that didn't matter at all. I agree that it is better to use the constant directly in the helper method. I updated it. I also executed the test locally against TS3 and it still works.